<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eye Tracking</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="ConsoleBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>980</width>
      <height>125</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Console</string>
    </property>
    <widget class="QTextEdit" name="Console">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>960</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ScreenBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>660</width>
      <height>510</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Screen</string>
    </property>
    <widget class="QWidget" name="Screen" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>640</width>
       <height>480</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="SettingsBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>310</width>
      <height>510</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QSlider" name="ExposureSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>480</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>48</number>
     </property>
    </widget>
    <widget class="QSlider" name="ThresholdSlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>25</number>
     </property>
    </widget>
    <widget class="QSlider" name="IntensitySlider">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="pageStep">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Intensity</string>
     </property>
    </widget>
    <widget class="QComboBox" name="CameraModesCBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>180</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Segment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Object</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Precision</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Grayscale</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MJPEG</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera Modes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AEC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic Exposure Control</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="AGC">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Automatic Gain Control</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionShut_Down"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSave_Frame_As_an_Image"/>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionShut_Down">
   <property name="text">
    <string>Shut Down</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionSave_Frame_As_an_Image">
   <property name="text">
    <string>Save Frame As an Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
